src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Media/ImageBrushTests/ImageBrush_Shape_Fill_StretchAndAlignment.TestCases.cs

```diff
@@ -0,0 +1,1558 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SamplesApp.UITests.Windows_UI_Xaml_Media.ImageBrushTests
+{
+	partial class ImageBrush_Shape_Fill_StretchAndAlignment
+	{
+		private static ExpectedColor[] GetExpectedColors()
+		{
+			return new[]
```

Comment:
Yeah I wanted to but I wasn't sure how to tackle it since each permutation might have a different set of expected colors so I didn't know how to do it in a generic sort of way